state/remote/atlas.go

```diff
@@ -48,7 +48,7 @@ func atlasFactory(conf map[string]string) (Client, error) {
 
 	parts := strings.Split(name, "/")
 	if len(parts) != 2 {
-		return nil, fmt.Errorf("malformed name '%s'", name)
+		return nil, fmt.Errorf("malformed name '%s' (valid format is 'username/name')", name)
```

Comment:
I'll let @mitchellh chime in, but I think the confusion is this:

- If you are using Atlas remote state, it needs to be `username/name` since that's what Atlas expects.
- If you are using a different remote state (like S3 for example), the name is just `name` - it can be more generic.

Does that make sense?